buddyboss/buddyboss-platform
# message media privacy fixed

### All Submissions:






### Changes proposed in this Pull Request:



Fixes #388 

### How to test the changes in this Pull Request:

1. Go to member's message compose screen and upload and add media in message
2. Send the message and see the profile > photos
3. Photos uploaded to message should not appear in the photos list.

### Other information:




### Changelog entry

> Messages - Privacy of media uploaded to message fixed.
